doc/config.rst

```diff
@@ -71,3 +71,19 @@ configure them in your config file.
         ->setIndent("\t")
         ->setLineEnding("\r\n")
     ;
+
+It's possible to register custom rule sets, which makes it easier to reuse custom configuration between multiple projects. If you have prepared rule set, you can register it and then enable it in the rules. Custom rule sets (in this example `\MyNameSpace\MyRuleSetClass`) must implement `\PhpCsFixer\RuleSet\RuleSetDescriptionInterface`.
+
+.. code-block:: php
+
+    <?php
+
+    $config = new PhpCsFixer\Config();
+    return $config
+        ->registerCustomRuleSets([
+            '@MyRuleSet' => \MyNameSpace\MyRuleSetClass::class,
+        ])
+        ->setRules([
+            '@MyRuleSet' => true,
+        ])
+    ;
```

Comment:
```suggestion
    ;

ℹ️ If you use other `\PhpCsFixer\ConfigInterface` implementation than built-in one, you can register custom rule sets using `\PhpCsFixer\RuleSet\RuleSets::registerRuleSet()` just before the configuration. Currently `\PhpCsFixer\Config:: registerCustomRuleSets()` is not part of the `ConfigInterface` because it would be a BC-break - it will be added there in next major version of the Fixer.
```

Reply:
Done